test/test_other.py

```diff
@@ -12872,10 +12872,16 @@ def test_extended_const(self):
 
   # Smoketest for MEMORY64 setting.  Most of the testing of MEMORY64 is by way of the wasm64
   # variant of the core test suite.
+  @parameterized({
+    'O0': (['-O0'],),
+    'O1': (['-O1'],),
+    'O2': (['-O2'],),
+    'O3': (['-O3'],),
+    'Oz': (['-Oz'],),
+  })
   @requires_wasm64
-  def test_memory64(self):
-    for opt in ['-O0', '-O1', '-O2', '-O3']:
```

Comment:
`-O3` was here before, so didn't the test pass already in the mode that the PR title says it fixes?

Reply:
The simple hello world program that was used here before did not trigger the calling of any native functions that need signature re-writing.   The one that was in the original bug report was stackAlloc, and that is triggered by the use of argv in the test code.   The old hello world program didn't use argv at all and so didn't include a call to stackAlloc.

Reply:
I see, thanks. lgtm then, with the suggestions applied.